tests/Makefile

```diff
@@ -4,8 +4,11 @@ test-integration:
 test-integration-cov:
 	go test ./integration/... -timeout 30m -coverpkg=../... -coverprofile=integration-profile.out -covermode=atomic
 
-test-e2e:
-	go test ./e2e/... -mod=readonly -timeout 30m -race -tags='e2e'
+test-e2e: test-e2e-server
+	go test ./e2e/... -mod=readonly -timeout 30m -tags='e2e'
 
 test-e2e-cov:
```

Comment:
```suggestion
test-e2e-cov: test-e2e-server
```

Reply:
I think we should still have that, and as well run it with coverage for our SonarCloud integration.